lua/lsp-installer.lua

```diff
@@ -0,0 +1,10 @@
+require("nvim-lsp-installer").settings {
```

Comment:
rare icon displayed when some **lsp languages** are installed, need to find out how to hide